### PR TITLE
add wiki link for user data config settings in teletraan UI page

### PR DESCRIPTION
**Pull request by lixmgl**

### CODE REVIEW

File: deploy-board/deploy_board/templates/clusters/cluster_configuration.html [hunk 1]
```diff
@@ -70,6 +70,7 @@ <h4 class="panel-title pull-left">Capacity</h4>
                         <div class="form-group"></div>
                         <div class="form-group">
                             <label class="deployToolTip control-label col-xs-2" data-toggle="tooltip"  title="Customize user data">User data configs:</label>
+                            <a href="https://w.pinadmin.com/pages/viewpage.action?spaceKey=IN&title=Launching+a+new+service#Launchinganewservice-DeploytheservicetoCMP/Dockercluster">wikiHelp</a>
```
Comment (yongwen) [inline deploy-board/deploy_board/templates/clusters/cluster_configuration.html]: as it is open source, please add pinterest specific link to env variable, same as other .pinadmin.com url.

Comment (lixmgl) [reply deploy-board/deploy_board/templates/clusters/cluster_configuration.html]: Thanks for reminding, updated.

File: deploy-board/deploy_board/templates/configs/new_capacity_adv.html [hunk 2]
```diff
@@ -56,6 +56,7 @@ <h4 class="panel-title pull-left">Capacity</h4>
                         <div class="form-group"></div>
                         <div class="form-group">
                             <label class="deployToolTip control-label col-xs-2" data-toggle="tooltip" title="Customize user data">User data configs:</label>
+                            <a href="{{user_data_config_settings_wiki}}">wikiHelp</a>
```
Comment (yongwen) [inline deploy-board/deploy_board/templates/configs/new_capacity_adv.html]: what if the env var is not set in non pinterest case? should we not show the wikihelp text?

Comment (yongwen) [reply deploy-board/deploy_board/templates/configs/new_capacity_adv.html]: maybe use the "?" icon instead to be consistent with other ui element

Comment (lixmgl) [reply deploy-board/deploy_board/templates/configs/new_capacity_adv.html]: Thanks for pointing out, updated.

File: deploy-board/deploy_board/templates/clusters/cluster_configuration.html [hunk 3]
```diff
@@ -70,6 +70,9 @@ <h4 class="panel-title pull-left">Capacity</h4>
                         <div class="form-group"></div>
                         <div class="form-group">
                             <label class="deployToolTip control-label col-xs-2" data-toggle="tooltip"  title="Customize user data">User data configs:</label>
+                            {% if is_pinterest %}
+                                <a href={{user_data_config_settings_wiki}}><span class="glyphicon glyphicon-question-sign"></span></a>
```
Comment (yongwen) [inline deploy-board/deploy_board/templates/clusters/cluster_configuration.html]: can you add target="_blank" after href so the wiki will show up in a new tab?

Comment (lixmgl) [reply deploy-board/deploy_board/templates/clusters/cluster_configuration.html]: good point, thanks, updated.

**Pull request merged**